### PR TITLE
Updated broken link

### PR DESCRIPTION
**Pull request by tilakrayal**
Updated the broken link for Sutskever et al., 2013 in documentation.

### CODE REVIEW

File: tensorflow/python/keras/optimizer_v2/gradient_descent.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ class SGD(optimizer_v2.OptimizerV2):
 
   Reference:
       - For `nesterov=True`, See [Sutskever et al., 2013](
-        http://jmlr.org/proceedings/papers/v28/sutskever13.pdf).
+        https://github.com/mlresearch/v28/blob/gh-pages/sutskever13.pdf).
```
Comment (bhack) [inline tensorflow/python/keras/optimizer_v2/gradient_descent.py:95]: http://proceedings.mlr.press/v28/sutskever13.pdf

**Comment by gbaned** [comment]
@tilakrayal Can you please check @bhack's comments. Thank you!

**Comment by gbaned** [comment]
@tilakrayal Any update on this PR? Please. Thank you!

**Comment by gbaned** [comment]
Hi @tilakrayal It looks like your PR relates to the Keras component. Please submit it to the github.com/keras-team/keras repository instead. Thankyou.
@fchollet, @qlzh727

**Pull request closed without merging**